pkg/template/template.go

```diff
@@ -50,7 +50,7 @@ func SetContextToURLQuery(u *url.URL, context map[string]interface{}) error {
 		return err
 	}
 	query := u.Query()
-	query.Set("x-skygear-redirect-data", encoded)
+	query.Set("x-authgear-redirect-data", encoded)
```

Comment:
This function `SetContextToURLQuery` is unused now. Its friends `DecodeURLQueryParamValueToContext` and `EncodeContextToURLQueryParamValue` can also be removed.